index.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const config  = require('./config');
+const request = require('superagent');
+const redact  = (key, value) => /password/i.test(key) ? value.split('').map(() => '*').join('') : value;
+
+// ServiceNow request
+module.exports = request
+  .post(config.endpoint)
+  .auth(config.username, config.password)
+  .type('application/json')
+  .set('Accept', 'application/json')
+  .send(config.message)
+  .then(res => {
+    console.log('Notification successfully sent, response code:', res.statusCode);
```

Comment:
We can probably be a bit more defensive here. i.e. We always expect a 201.

Reply:
And if we are opening a change we require an internal_id back. (Is that distinction for a later PR?)

Reply:
Yes, it was/is, but has now been sorted